src/index.css

```diff
@@ -89,13 +98,18 @@ line-height: 24px;
   margin-top: 8px;
 }
 
-@media only screen
-     and (width: 430px) 
-     and (device-height: 932px)
-     and (-webkit-device-pixel-ratio: 3) {
+@media (min-width: 431px) and (max-width: 834px){
+  h1{
+    margin-left: 24px; 
+    font-size: 48px;
+    line-height: 56px;
+    }
+}
 
-      h1{
-        font-size: x-large;
-        color: red;
-      }
+@media (min-width: 835px){
+  h1{
+    margin-left: 32px;
+    font-size: 48px;
+    line-height: 56px;
+  }
```

Comment:
Although, actually I notice something: This is the desktop CSS. However, you're already applying it for viewports 1px wider than the tablet frame in Figma.

The frames in Figma usually represent the smallest viewport widths. So, in this project:
* Mobile CSS would be applied up to 833px
* Tablet CSS would be applied from 834px to up to 1511px
* Desktop CSS would be applied from 1512px

Currently, you're applying it like this:
* Mobile CSS is applied up to 430px
* Tablet CSS is applied from 431px to up to 834px
* Desktop CSS is applied from 835px